deploy/k8s/manifest.yaml

```diff
@@ -375,7 +383,38 @@ data:
               - action: update_label
                 label: owner_name
                 new_label: deployment
-
+          - include: k8s.container_fs_reads_total
+            action: insert
+            match_type: regexp
+            # non-empty container, datapoints of Pod's file system read ops
+            experimental_match_labels: { "container": "(.|\\s)*\\S(.|\\s)*" }
+            new_name: k8s.pod.fs.reads.rate
+          - include: k8s.container_fs_writes_total
+            action: insert
+            match_type: regexp
+            # non-empty container, datapoints of Pod's file system write ops
+            experimental_match_labels: { "container": "(.|\\s)*\\S(.|\\s)*" }
+            new_name: k8s.pod.fs.writes.rate
+          - include: k8s.container_fs_reads_bytes_total
+            action: insert
+            match_type: regexp
+            # non-empty container, datapoints of Pod's file system read bytes
+            experimental_match_labels: { "container": "(.|\\s)*\\S(.|\\s)*" }
+            new_name: k8s.pod.fs.reads.bytes.rate
+          - include: k8s.container_fs_writes_bytes_total
+            action: insert
+            match_type: regexp
+            # non-empty container, datapoints of Pod's file system written bytes
+            experimental_match_labels: { "container": "(.|\\s)*\\S(.|\\s)*" }
+            new_name: k8s.pod.fs.writes.bytes.rate
+          - include: k8s.container_fs_usage_bytes
+            action: insert
+            match_type: regexp
```

Comment:
what is the reason of `match_type: regexp`?

Reply:
Fixed